csv/main.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./csvreader-config";
```

Comment:
Does it work like this? Would be perfect if it does.

Reply:
yes, it works